clock/scheduledJobs/deactivateRooms.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * clock/scheduledJobs/deactivateRooms.js
+ *
+ * Gets an array of all rooms which should be deactivated, then deactivates
+ * them and does a sync if this is enabled in the room settings.
+ */
+const featureToggles = require('feature-toggles');
+const moment = require('moment');
+const Op = require('sequelize').Op;
+
+const dbRoom = require('../../db/index').Room;
+const dbEvent = require('../../db/index').Event;
+const dbBotAction = require('../../db/index').botAction;
+
+const TWO = 2;
+const THRESHOLD_IN_MINUTES = 120; // 2 hours
```

Comment:
should be coming from config - 2 hrs might seem like a reasonable duration now, but who knows six months from now - you can hard-code a reasonable default in the config file but allow admin to override using env var